releasePipeline/20-build-galasa.yaml

```diff
@@ -22,7 +22,7 @@ spec:
     value: release-maven-repos
   - name: jacocoEnabled
     value: "false"
-  - name: isRelease
+  - name: isMainOrRelease
```

Comment:
Is isMainOrRelease ever set to "false" ?

Reply:
Yes it should be for branch builds so if I wanted a development maven repository for my branch 'jade' and I did a branch build for that, I'd set isMainOrRelease to false then. It also should be false on any local builds, so isMainOrRelease=false in the gradle.properties for each repo.